R/drive_search.R

```diff
@@ -47,6 +52,13 @@ drive_search <- function(pattern = NULL, ..., verbose = TRUE) {
     params$fields <- paste0("files/", .drive$default_fields, collapse = ",")
   }
 
+  if (!is.null(type)) {
+    mime_type <- drive_mime_type(type, verbose = verbose)
```

Comment:
Oh wait, no you can't! This is the other way around, where you want to preserve `NULL`.